docs/src/main/asciidoc/cdi-integration.adoc

```diff
@@ -10,9 +10,10 @@ include::_attributes.adoc[]
 :numbered:
 :toclevels: 2
 
-ArC, the CDI container, is bootstrapped at build time.
-The downside of this approach is that CDI Portable Extensions cannot be supported.
-Nevertheless, the functionality can be achieved using the Quarkus-specific extensions API.
+ArC, the CDI container in Quarkus, is bootstrapped at build time.
+To integrate with the container, CDI Build Compatible Extensions can be used, as well as a Quarkus-specific extension API.
+CDI Portable Extensions are not and cannot be supported.
+This guide focuses on the Quarkus-specific extensions API.
```

Comment:
Maybe add a [link](https://jakarta.ee/specifications/cdi/4.0/jakarta-cdi-spec-4.0.html#spi_lite) to the Jakarta spec as a source of information for BCE?

Reply:
Good idea!

Reply:
Done.